### PR TITLE
[WNMGDS-852] Adding controlled accordion 

### PR DESCRIPTION
**Pull request by line47**
## Summary
This PR adds the ability to for controlling the accordion component

## How to test
Go to the demo site accordion page 
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-852-accordion-control/components/accordion/

Click the react examples and ensure that the accordions open and close independently.

### CODE REVIEW

File: packages/design-system-docs/src/pages/components/Accordion/Accordion.example.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      </>
+    );
+  }
+}
+
+ReactDOM.render(
+  <div className="example--wrapper">
+    <ControlledAccordion />
+
+    <Accordion bordered variation="single">
```
Comment (kimieann) [inline packages/design-system-docs/src/pages/components/Accordion/Accordion.example.jsx:59]: I don't think variation is needed here anymore.

**Review by kimieann — COMMENTED** [review]
Just one comment, but everything else looks good!

**Review by nichia — COMMENTED** [review]
Just some errors from running `yarn test` to be fixed. Other than that, it looks good.

**Review by nichia — APPROVED** [review]

**Review by kimieann — APPROVED** [review]

**Comment by line47** [comment]
@nichia to aadd guidance on how to use the controlled version of the accordion component

**Comment by line47** [comment]
@nichia and @kimieann I think this is good to merge - what do you all think?

**Comment by kimieann** [comment]
> @nichia and @kimieann I think this is good to merge - what do you all think?

i'm good with it!

**Pull request merged**